Source: config-top.h
Upstream: Yes.
Reason: We want a global bashrc. This merely uncomments that define.

diff -ur bash-3.2-orig/config-top.h bash-3.2/config-top.h
--- bash-3.2-orig/config-top.h	2005-04-29 16:36:34.000000000 -0400
+++ bash-3.2/config-top.h	2007-11-01 19:53:38.000000000 -0400
@@ -73,7 +73,7 @@
 #define KSH_COMPATIBLE_SELECT
 
 /* System-wide .bashrc file for interactive shells. */
-/* #define SYS_BASHRC "/etc/bash.bashrc" */
+#define SYS_BASHRC "/etc/bash/bashrc"
 
 /* System-wide .bash_logout for login shells. */
 /* #define SYS_BASH_LOGOUT "/etc/bash.bash_logout" */
